.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@auth0/babel-preset-cosmos"]
```

Comment:
I don't think you should drop this into root, that can lead to all the other internal configs falling back to this